flame-demo/README.md

```diff
@@ -0,0 +1,30 @@
+# testapp-flutter
```

Comment:
Did you copy this over? I think it deserves its own README

Reply:
I absolutely did copy this over lol

Yeah, definitely; I'll do that as part of the next batch of tasks.